### PR TITLE
:memo: Add  `net_url/_urls.ts`

### PR DESCRIPTION
**Pull request by TaiHusk**
Hello! I'm submitting this pull request as my contribution to the open-source community and to help improve your project. I've made several changes focused on enhancing code readability and efficiency. Below, I've explained the rationale behind each modification.

Introduction of `net_url/_urls.ts` file:
I created the `net_url/_urls.ts` file to address the following concerns:
- As the project grows, maintaining strict source code rules for large projects becomes increasingly important.
- Centralizing URL management saves time and simplifies the process of correcting paths across the site.

### CODE REVIEW

**Review by mattgodbolt — CHANGES_REQUESTED** [review]
This contains a number of unrelated changes. For example

- you delete the `hello-nasm.asm` file in this change, which doesn't seem related to `_urls.ts`
- you change the README.md.
- you have added extra whitespace in a number of places

This appears to be the second time you've opened a fairly similar PR, and you haven't responded to the issues raised by the reviewers in the other PR.

We appreciate you wishing to help us, but this is not the way to do so. Please, take a look at this PR on Github, and see:

- all the unrelated differences in style
- all the failing tests
- the failing pre-commit hooks

I'm sorry to dampen your enthusiasm but we can't merge PRs like this.

If you want to continue, please open small, targeted PRs to (for example) update the README on its own. Install and run the `pre-commit` hooks, and ensure they pass. Run the tests, if you can't get to the pre-commit working. run `npm run lint` and so on. Then submit a PR, and see how it goes.

File: net_url/_urls.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            quickbench: 'https://quick-bench.com',
+            cloogle: 'https://cloogle.org',
+            cppinsights: 'https://cppinsights.io',
+            static_ce_cdn: 'https://static.ce-cdn.net',
+            github: 'https://github.com',
+        },
+
+        HTTP: {
+            reddit: 'http://www.reddit.com',
+        },
```
Comment (junlarsen) [inline net_url/_urls.ts:14]: we already went over this in the other PR. we should instead just export each url as its own constant. there is no reason to introduce a class

**Review by junlarsen — CHANGES_REQUESTED** [review]
Not sure about the filename, it's only used on the frontend, so I think something like `/static/urls.ts` would be more appropriate

**Comment by mattgodbolt** [comment]
Closing: sorry we couldn't find a way to make this work.

**Pull request closed without merging**